ext/pdo_mysql/tests/bug78152.phpt

```diff
@@ -4,31 +4,32 @@ Bug #78152: PDO::exec() - Bad error handling with multiple commands
 pdo_mysql
 --SKIPIF--
 <?php
-require_once(dirname(__FILE__) . DIRECTORY_SEPARATOR . 'mysql_pdo_test.inc');
+require_once __DIR__ . '/inc/mysql_pdo_test.inc';
 MySQLPDOTest::skip();
 ?>
 --FILE--
 <?php
-
-require_once(dirname(__FILE__) . DIRECTORY_SEPARATOR . 'mysql_pdo_test.inc');
+require_once __DIR__ . '/inc/mysql_pdo_test.inc';
 $db = MySQLPDOTest::factory();
-MySQLPDOTest::createTestTable($db);
 
-var_dump($db->exec("INSERT INTO test(id, label) VALUES (41, 'x'); INSERT INTO test_bad(id, label) VALUES (42, 'y')"));
+$table = 'bug78152_pdo_mysql';
+MySQLPDOTest::createTestTable($table, $db);
+
+var_dump($db->exec("INSERT INTO {$table} (id, label) VALUES (41, 'x'); INSERT INTO {$table}_bad (id, label) VALUES (42, 'y')"));
 $db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
 try {
-    var_dump($db->exec("INSERT INTO test(id, label) VALUES (42, 'x'); INSERT INTO test_bad(id, label) VALUES (43, 'y')"));
+    var_dump($db->exec("INSERT INTO {$table} (id, label) VALUES (42, 'x'); INSERT INTO {$table}_bad (id, label) VALUES (43, 'y')"));
 } catch (PDOException $e) {
     echo $e->getMessage(), "\n";
 }
-
```

Comment:
The warning version feels redundant? But that's by the by

Reply:
In case of ERRMODE, it is not possible to check the return value of `exec()